helm_deploy/values-dev.yaml

```diff
@@ -11,8 +11,3 @@ generic-service:
     HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     PRISON_API_URL: https://prison-api-dev.prison.service.justice.gov.uk
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
-
-# CloudPlatform AlertManager receiver to route promethues alerts to slack
-# See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
-generic-prometheus-alerts:
```

Comment:
Also was this removed on purpose, didn't see this. Could it go to visits-alerts-nonprod

Reply:
@adaviesMOJ it's now in the default `values.yaml` file 👍

Reply:
Only prod actually needs to go to the `vists-alerts` prod channel